packages/i18n/src/index.js

```diff
@@ -34,7 +35,9 @@ export function setLocaleData( localeData = { '': {} }, domain = 'default' ) {
 		} );
 	}
 
-	i18n.options.locale_data[ domain ] = localeData;
+	i18n.options.locale_data[ domain ] = has( i18n.options.locale_data, domain )
```

Comment:
`has` is a deceptively simple function, one which as used here will invoke a very complex chain of calls to parse the `domain` argument as a string path (and thus include in the module itself). I think we might be better off using `Object#hasOwnProperty` here.

Reply:
ooo did not know that, good to know.  I thought has might be different than `hasIn` and `hasIn` had the behaviour you described.  I'll fix.